IHP/View/CSSFramework.hs

```diff
@@ -170,28 +170,29 @@ instance Default CSSFramework where
             styledSubmitButtonClass = ""
 
             styledPagination :: CSSFramework -> PaginationView -> Blaze.Html
-            styledPagination _ paginationView =
-                [hsx|
+            styledPagination _ paginationView@PaginationView {pagination, linkPrevious, linkNext, pageDotDotItems, itemsPerPageSelector} =
+                when (showPagination pagination)
```

Comment:
IMO it would be better to put the view logic here into [`renderPagination`](https://github.com/digitallyinduced/ihp/blob/master/IHP/Pagination/ViewFunctions.hs#L30). The CSSFramework should only deal with very basic html generation / styling.

Reply:
Good point :+1: